src/libs/ValidationUtils.js

```diff
@@ -456,6 +481,37 @@ function isValidAccountRoute(accountID) {
     return CONST.REGEX.NUMBER.test(accountID) && accountID > 0;
 }
 
+const isTimeAtLeastOneMinuteInFuture = (inputTime = '', inputDateTime = '') => {
+    if (!inputTime) return false;
+    // Parse the hour and minute from the time input
+    const [hourStr, minuteStr, period] = inputTime.split(/[:\s]+/);
+    let hour = parseInt(hourStr, 10);
+
+    // Convert 12-hour format to 24-hour format
+    if (period.toUpperCase() === 'PM' && hour !== 12) {
+        hour += 12;
+    } else if (period.toUpperCase() === 'AM' && hour === 12) {
+        hour = 0;
+    }
+
+    const minute = parseInt(minuteStr, 10);
+
+    // Check if the inputDateTime contains a time portion or is just a date
+    const dateTimeFormat = inputDateTime.includes(':') ? 'yyyy-MM-dd HH:mm:ss' : 'yyyy-MM-dd';
+
+    // Parse the input date and time (or just date) accordingly
+    const parsedInputDateTime = parse(inputDateTime, dateTimeFormat, new Date());
+
+    // Set the hour and minute from the time input to the date from the date-time input
+    const combinedDate = set(parsedInputDateTime, {hours: hour, minutes: minute});
+
+    // Get current date and time
+    const now = new Date();
+
+    // Check if the combinedDate is at least one minute later than the current date and time
+    return isAfter(combinedDate, addMinutes(now, 1));
```

Comment:
I see that we're using `date-fns` here and `moment` elsewhere, just wondering how easy it would be to drop moment since it's a larger package. Just a thought rather than a request as I know we're just trying to get this finished!

Reply:
It would be great to ditch the moment. When I started this PR, I didn't see we had begun this process. We need to point it up in the doc somewhere.